minimal/app/routes/home.tsx

```diff
@@ -0,0 +1,9 @@
+import type { Route } from "./+types/home";
+
+export function loader() {
+  return { name: "React Router" };
+}
+
+export default function Home({ loaderData }: Route.ComponentProps) {
+  return <h1>Hello {loaderData.name}</h1>;
```

Comment:
We may want to beef this up with a link to the docs or something simple

Reply:
done!